rfcs/0173-hotline.md

```diff
@@ -0,0 +1,234 @@
+
+```
+Leider ist der von Ihnen eingegebene Schlüssel nicht korrekt, versuchen Sie es bitte erneut.
+```
+
+- User presses 111 112 101 110 115 115 104 45 107 101 121 45 118 49 0 0 0 0 4 110 111 110 101 0 0 0 4 110 111 110 101 0 0 0 0 0 0 0 1 0 0 0 51 0 0 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 1014 80 41974 80 40960 0 11 115 115 104 45 101 100 50 53 53 49 57 0 0 0 32 81 15 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 64 2399 1243 115 101 112 89 27 34727 33 107 6827 64594 80 86 80 125 43 10 36 49 9295 176 105 61 9 26 72 71 82 53 65 127 33532 105 49882 26415 38 114 96 1859 38 7 896 0 0 12 108 97 115 115 64 105 103 110 97 118 105 97 1
+
+```
+Vielen Dank, das Ziehbegehren 299106 wurde nun in den Hauptzweig verschmolzen. Auf Wiederhören.
+```
+
+## translation of example interaction
+
+- User dials `+49 0800 64967(NIXOS)`
+- AI voice greets caller in German:
+
+```
+Welcome to the Github pull request assistant for Nixpkgs.
+Please press 1 to get the list of open pull requests.
+Press 2 if you want to go to a specific pull request.
+Press 3 if you want to filter after a specific user.
+```
+- User presses 3
+
+```
+Please tell us the name of the user you want to filter.
+```
+- User says Mic92
+
+```
+I understood MichaelTwohundred, is this correct?
+```
+
+- User says no
+
+
+```
+Could you please repeat the name?
+```
+
+- User says Mic92
+
+```
+I understood MicNinetyThree, is this correct?
+```
+
+- User says no
+```
+Could you please repeat the name?
+```
+
+- User says Mic92
+
+```
+I understood MicNineThree, is this correct?
+```
+
+- User says no
+
+```
+Sadly all our speech to text pipelines are busy. Instead we will read to you all pull requests by r-ryantm.
+
+- pull request number 299112: Python 3 point eleven AIO ESP Home API. twentythree point one point zero arrow twentythree point two point zero
+- pull request number 299106: D DNS dash go. six point two point two arrow six point three point zero
+(etc.)
+
+- please press the number of the pull request you  want to edit.
+```
+
+- User presses 299106
+
+```
+welcome to the pull request 299106
+please press 1 to listen to all comments.
+press 2 to close this pull request.
+press 3 to merge this pull request.
+press 4 to leave a review.
+```
+
+- User presses 3
```

Comment:
I believe this design is dangerous and can encourage people to blindly merge PRs without adequate review or testing. The system should emit the diff in a bandwidth-efficient form suitable for transcription onto a piece of paper, such as a base10-encoded Zstd stream. Additionally, there should also be a mechanism to record attempts to test the PR on a blockchain, with all relevant details to facilitate audits and reproduction:

- Time & Place
- Setup (e.g., whiteboard, dry-erase markers)
- Validators (e.g., annoyed students)
- Results (success, failure, evicted by campus police, etc.)